content/applications/hr/recruitment/source_analysis.rst

```diff
@@ -0,0 +1,81 @@
+=========================
+Source analysis reporting
+=========================
+
+Reporting is a critical tool for any recruiting department, to gain insights into the entire
+recruitment process.
+
+Determining where applicants come from can inform on which sources provide the best results. This
+information is determined by the *Source Analysis* report. With this data, recruitment teams can
+better pivot their recruiting strategies, in an attempt to gain better applicants, in both quantity
+and quality.
+
+Open report
+===========
+
+To access the *Source Analysis* report, navigate to :menuselection:`Recruitment app --> Reporting
+--> Source Analysis`. This presents the data for the :icon:`fa-filter` :guilabel:`Last 365 Days
+Applicant`, in a default :icon:`fa-area-chart` :guilabel:`Graph` view, showing the amount of
+applicants by :guilabel:`Source`, and further separated by stage (:guilabel:`In Progress` and
+:guilabel:`Hired`). Hover the cursor over any column to view the specific numbers fort that column.
```

Comment:
```suggestion
To access the *Source Analysis* report, navigate to :menuselection:`Recruitment app --> Reporting
--> Source Analysis`. 

This presents the data for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`, in a default
:icon:`fa-area-chart` :guilabel:`Graph` view, showing the amount of applicants by :guilabel:`Source`,
and further separated by stage (:guilabel:`In Progress` and :guilabel:`Hired`).

Hover the cursor over any column to view the specific numbers for that column.
```
OPTIONAL way to break up this otherwise "lengthy" paragraph